src/main/java/org/junit/runners/ParentRunner.java

```diff
@@ -358,6 +358,7 @@ public Description getDescription() {
     public void run(final RunNotifier notifier) {
         EachTestNotifier testNotifier = new EachTestNotifier(notifier,
                 getDescription());
+        testNotifier.fireTestSuiteStarted();
```

Comment:
Note that `ParentRunner` is the base class of `Parameterized`. I'm not sure if people would find the events sent from `Parameterized` confusing